satellite/rewards/rewards.go

```diff
@@ -69,12 +69,13 @@ const (
 type OfferStatus int
 
 const (
-	// Done is a default offer status when an offer is not being used currently
-	Done = OfferStatus(iota)
+
+	// Active is a offer status when an offer is currently being used
+	Active = OfferStatus(iota)
 	// Default is a offer status when an offer is used as a default offer
 	Default
-	// Active is a offer status when an offer is currently being used
-	Active
+	// Done is a default offer status when an offer is not being used currently
```

Comment:
The comment here is not reflecting what's happening. `Done` is not default anymore

Reply:
fixed